test/CMakeLists.txt

```diff
@@ -293,3 +293,6 @@ add_custom_target(process_test_results
                   pandoc -s -o TEST_RESULTS.html --metadata title="Enzo-E Test Results" TEST_RESULTS.md
                   WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
 
+# Load Balance
+#setup_test_parallel(LoadBalance-1 LoadBalance/morton-1  input/LoadBalance/test_balance-on-bcg.in)
```

Comment:
The test for the Morton method was actually already removed before I added the Hilbert method. My guess is the original test was removed because it was a lengthy cosmological simulation. I added it back in temporarily while I was putting together the Hilbert test and then commented it out. I didn't want to reintroduce a test which was removed and I probably should have removed it entirely instead of commenting it out.